listenbrainz/webserver/templates/index/playlists-management.html

```diff
@@ -0,0 +1,34 @@
+{%- extends 'base.html' -%}
+{%- block title -%}Playlists management - ListenBrainz{%- endblock -%}
+{%- block content -%}
+  <h2 class="page-title">Playlists managementg </h2>
```

Comment:
```suggestion
  <h2 class="page-title">Playlist management </h2>
```

This might end up renamed as "Importing playlists", but thought I'd comment in any case.